### PR TITLE
Inventory file extensions

### PR DESCRIPTION
**Pull request by RichardHoch**
Change to text of Migration Analytics Guide, Chapter 5.1, "Creating and filtering a Migration Analytics Report"
Added .zip and .json files to the inventory extensions supported by the Migration Analytic Report service. This meant changing the text of the Prerequisites and step 3 of the procedure.

### CODE REVIEW

**Comment by apinnick** [comment]
If this is ready for review, could you please create and upload a doc preview? I realise that it's not necessary for such a small change, but I would like you to try.

**Comment by RichardHoch** [comment]
Preview of section: http://file.emea.redhat.com/~rhoch/#Creating-migration-analytics-report_migration_analytics

**Comment by apinnick** [comment]
> Preview of section: http://file.emea.redhat.com/~rhoch/#Creating-migration-analytics-report_migration_analytics

Please delete this preview and create the correct directory structure: https://mojo.redhat.com/docs/DOC-998241

**Comment by apinnick** [comment]
Also, this should be a draft pull request, not a pull request. I don't think you can change it now.

**Comment by apinnick** [comment]
Also, remember to put the Jira in the PR description. All relevant links should go there so that people can check the wording of the task or bug before they review it.

**Comment by RichardHoch** [comment]
Preview of proposed change: http://file.emea.redhat.com/~rhoch/ma_reports/#Creating-migration-analytics-report_migration_analytics

JIRA: https://issues.redhat.com/browse/MIGENG-503

**Comment by RichardHoch** [comment]
@apinnick  Please comment on the proposed changes to the published text:

Prerequisite item:

Current text: Inventory data file generated by Red Hat CloudForms or a `tar.gz` archive containing multiple inventory data files

Proposed: Inventory data file generated by Red Hat CloudForms or an archive containing one or more inventory data files (supported types:`tar.gz,` '.zip' and 'json')

Mandatory fields:

Current text: ** Inventory data file: Inventory data file generated by CloudForms. Click Browse to select the `tar.gz` file.

Proposed 1: ** Inventory data file: Inventory data file generated by CloudForms. Click Browse to select the `tar.gz` file ('.zip' and '.json' format inventory data files can also be used). (Phil's request as is) .

Proposed 2: ** Inventory data file: Inventory data file generated by CloudForms or an archive containing one or more inventory data files. Click Browse to select the `tar.gz, .zip,` or `.json` file.

JIRA: https://issues.redhat.com/browse/MIGENG-503

**Comment by apinnick** [comment]
> @apinnick Please comment on the proposed changes to the published text:
> 
> Prerequisite item:
> 
> Current text: Inventory data file generated by Red Hat CloudForms or a `tar.gz` archive containing multiple inventory data files
> 
> Proposed: Inventory data file generated by Red Hat CloudForms or an archive containing one or more inventory data files (supported types:`tar.gz,` '.zip' and 'json')
> 
> Mandatory fields:
> 
> Current text: ** Inventory data file: Inventory data file generated by CloudForms. Click Browse to select the `tar.gz` file.
> 
> Proposed 1: ** Inventory data file: Inventory data file generated by CloudForms. Click Browse to select the `tar.gz` file ('.zip' and '.json' format inventory data files can also be used). (Phil's request as is) .
> 
> Proposed 2: ** Inventory data file: Inventory data file generated by CloudForms or an archive containing one or more inventory data files. Click Browse to select the `tar.gz, .zip,` or `.json` file.
> 
> JIRA: https://issues.redhat.com/browse/MIGENG-503

Please make the changes and push a commit. Then I will comment.

**Comment by RichardHoch** [comment]
I changed the text to now read as follows:

Inventory data file generated by Red Hat CloudForms or an archive containing inventory data files (supported types: `tar.gz,` `.zip,` or `.json`)
*: Inventory data file generated by CloudForms. Click *Browse* to select the `tar.gz` file (`.zip` or `.json` files containing inventory data can also be used)

@apinnick -- please review. 

JIRA: https://issues.redhat.com/browse/MIGENG-503

**Comment by RichardHoch** [comment]
@apinnick 

I made the following proposed changes:

Prerequisite item changed to:
*  A `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms

Descriptoin of Inventory data file changed to:
** *Inventory data file*: a `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms. Click *Browse* to select the `tar.gz` file.

Periods added to descriptions of mandatory and optional fields in the name of consistency (since at least one required a period).

JIRA: https://issues.redhat.com/browse/MIGENG-503

File: doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ You can create a report with the link:https://cloud.redhat.com/beta[Migration An
 
 .Prerequisites
 
-* Inventory data file generated by Red Hat CloudForms or a `tar.gz` archive containing multiple inventory data files
+*  A `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms
```
Comment (apinnick) [inline doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc:10]: ' A `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms'

Change to '`tar.gz` archive file containing one or more `.json` inventory data files'

I think that is sufficient.

Comment (apinnick) [reply doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc:10]: Please squash your commits: https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#heading=h.epes4xf06pzk

File: doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc [hunk 2]
```diff
@@ -20,12 +20,12 @@ The *Reports* window is displayed.
 . Click *Create* and fill in the following fields:
 
 * Mandatory fields:
-** *Inventory data file*: Inventory data file generated by CloudForms. Click *Browse* to select the `tar.gz` file.
-** *Report name*: Name of your report, up to 250 characters
-** *Year-over-year growth rate for new hypervisors*: Number of new hypervisors to be added each year
+** *Inventory data file*: a `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms. Click *Browse* to select the `tar.gz` file.
```
Comment (apinnick) [inline doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc:23]: I think we can remove 'a `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms. ' 

Just 'Click browse, etc.'

File: doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc (hunk 1 above)
Comment (RichardHoch) [reply doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc]: Done.

File: doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc (hunk 2 above)
Comment (RichardHoch) [reply doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc]: Done.

**Comment by RichardHoch** [comment]
Edited as per comments.

File: doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc [hunk 3]
```diff
@@ -7,7 +7,7 @@ You can create a report with the link:https://cloud.redhat.com/beta[Migration An
 
 .Prerequisites
 
-*  A `tar.gz` archive file containing one or more `.json` inventory data files generated by CloudForms
+*  `tar.gz` archive file containing one or more `.json` inventory data files
```
Comment (apinnick) [inline doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc:10]: There's an extra space here.

File: doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc [hunk 4]
```diff
@@ -20,12 +20,12 @@ The *Reports* window is displayed.
 . Click *Create* and fill in the following fields:
 
 * Mandatory fields:
-** *Inventory data file*: Inventory data file generated by CloudForms. Click *Browse* to select the `tar.gz` file.
-** *Report name*: Name of your report, up to 250 characters
-** *Year-over-year growth rate for new hypervisors*: Number of new hypervisors to be added each year
+** *Inventory data file*: `tar.gz` archive file containing one or more `.json` inventory data files. Click *Browse* to select the `tar.gz` file.
```
Comment (apinnick) [inline doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc:23]: Forgot to fix this. I guess a preview would have been a good idea after all.

`tar.gz` archive file containing one or more `.json` inventory data files. Click *Browse* to select the `tar.gz` file.

Please remove ``tar.gz` archive file containing one or more `.json` inventory data files.`

**Pull request merged**